server/test/unit/vitalsTest/checkButtonHeartbeatTest.js

```diff
@@ -41,6 +44,7 @@ describe('vitals.js unit tests: checkButtonHeartbeat', () => {
     sandbox.spy(helpers, 'log')
 
     this.sendNotificationStub = sandbox.stub()
+    vitals.__set__('sendNotification', this.sendNotificationStub)
```

Comment:
I wonder if you need this. Is `checkButtonHeartbeat` calling `sendNotification` at all?

Reply:
Ahh no, good catch! I originally added this when my PR was going to using the `sendNotification` function, but since I've changed it to just log a Sentry message instead, this is not needed. Thanks!